UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs

```diff
@@ -89,5 +89,51 @@ public bool CheckFileExists(string filePath)
         {
             return File.Exists(filePath);
         }
+
+        public byte[] ReadAllBytes(string filePath)
+        {
+            return File.ReadAllBytes(filePath);
```

Comment:
Can we check if fileexists before reading bytes data

Reply:
Modified. Resolved.